src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IProjectCapabilitiesService.cs

```diff
@@ -1,6 +1,5 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
```

Comment:
Can we remove the extra line between the header and the namespace for places where all the using are removed?

Reply:
Seems like something the analyzer/quick-fix should be doing. Bug?

Reply:
I'll file a bug.

Reply:
https://github.com/dotnet/roslyn/issues/23568